tests/fixtures/sample_token_uris.ts

```diff
@@ -33,9 +28,8 @@ const IMAGE_IS_IPFS_PINNING_SERVICE_RESPONSE = {
   attributes: [{ trait_type: "Pudgy", value: "true" }],
 };
 
-const IPFS_PINNING_SERVICE_IMAGE_IS_IPFS_PINNING_SERVICE_RESPONSE = {
-  data: IMAGE_IS_IPFS_PINNING_SERVICE_RESPONSE,
-};
+const IPFS_PINNING_SERVICE_IMAGE_IS_IPFS_PINNING_SERVICE_RESPONSE =
+  IMAGE_IS_IPFS_PINNING_SERVICE_RESPONSE;
```

Comment:
These might could be removed now that they're no longer wrapped in a `data` attribute by native fetch